ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -454,13 +456,12 @@ internal class CustomerCenterViewModelImpl(
                             pricingPhasesDescription,
                         ),
                     )
-                    return true
                 } else {
                     currentState
                 }
             }
         }
-        return false
+        return loaded
```

Comment:
I think I can refactor this to check if there's a `promotionalOfferData` in the state, but I would need to do more tests